src/OpenTelemetry/Resources/ResourceBuilderExtensions.cs

```diff
@@ -110,16 +110,16 @@ public static ResourceBuilder AddAttributes(this ResourceBuilder resourceBuilder
         }
 
         /// <summary>
-        /// Adds resource attributes parsed from an environment variable to a
-        /// <see cref="ResourceBuilder"/> following the <a
+        /// Adds resource attributes parsed from OTEL_RESOURCE_ATTRIBUTES, OTEL_SERVICE_NAME environment variables
+        /// to a <see cref="ResourceBuilder"/> following the <a
         /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md#specifying-resource-information-via-an-environment-variable">Resource
         /// SDK</a>.
         /// </summary>
         /// <param name="resourceBuilder"><see cref="ResourceBuilder"/>.</param>
         /// <returns>Returns <see cref="ResourceBuilder"/> for chaining.</returns>
         public static ResourceBuilder AddEnvironmentVariableDetector(this ResourceBuilder resourceBuilder)
```

Comment:
Couldn't this detector be out-of-the-box? This is what other languages do.

_For sure something for a separate PR and maybe even needs some GH Issue for discussion._

Reply:
+1 on making this out-of-the-box

Reply:
I will make a separate PR to address it